bin/seed_janus

```diff
@@ -0,0 +1,30 @@
+#! /usr/bin/env bash
```

Comment:
Just a convenience script for quickly adding a useful developer administrator to janus db.  Used after cleaning my janus db from some other testing.